pkg/settlement/pseudosettle/mock/pseudosettle.go

```diff
@@ -23,6 +23,8 @@ type Service struct {
 
 	settlementsSentFunc func() (map[string]uint64, error)
 	settlementsRecvFunc func() (map[string]uint64, error)
+
+	pay func(context.Context, swarm.Address, uint64) error
```

Comment:
For consistency, this `pay` field should be called `payFunc`.

Reply:
fixed